.github/workflows/tests.yml

```diff
@@ -16,10 +16,10 @@ jobs:
         - Windows
         - MacOS
         py:
-        - 3.5
-        - 3.6
-        - 3.7
         - 3.8
+        - 3.7
+        - 3.6
+        - 3.5
```

Comment:
Minor point, but this is unrelated so I would have preferred a separate PR.

Reply:
Removed 🤷